wallet/src/database/model/WalletModel.ts

```diff
@@ -26,10 +26,10 @@ export class WalletModel extends Model<WalletModelAttributes> {
       type: DataTypes.STRING,
       allowNull: false,
     },
-    rapyd_ewallet_currency: {
+    rapyd_ewallet_currency_code: {
       type: DataTypes.STRING,
     },
-    rapyd_ewallet_country: {
+    rapyd_ewallet_country_code: {
```

Comment:
@Ktoxcon 
I renamed the fields to have the `*_code` suffix. 

The specificity is important here, since by having only `currency` and `country` implies we'll get a whole object in the WalletDAO, when in fact, we only get the country and currency `code`.